src/datastore/pg-write-store.ts

```diff
@@ -2329,7 +2349,7 @@ export class PgWriteStore extends PgStore {
     const updatedRows = await sql<{ tx_id: string }[]>`
       WITH restored AS (
         UPDATE mempool_txs
-        SET pruned = FALSE
+        SET pruned = FALSE, status = ${DbTxStatus.Pending}
```

Comment:
I think this may have fixed an super long standing bug where after a series of re-orgs and dropped mempool txs, a mempool tx could "disappear" because this "restore" function wouldn't reset the `status` back to pending.

Reply:
ahh man, great catch